isyfact-standards-doc/src/docs/antora/modules/blaupausen/pages/detailkonzept-komponente-service/inhalt.adoc

```diff
@@ -14,23 +14,29 @@ Diese technische Komponente besteht aus zwei Teilen: dem Service-Framework und d
 
 Das *Service-Framework* bezeichnet die konkrete Technologie, mit der die Services des Anwendungskerns zur Verfügung gestellt werden.
 
-Die *Servicelogik* enthält Daten und Funktionalität, die für die Bereitstellung des Services relevant sind.
-
+Die Fachkomponenten der *Servicelogik* Servicelogik beinhalten Service-Endpoints, 
+die über das Service-Framework bei eingehenden (Technologie-spezifischen) Requests aufgerufen werden. 
+Die Service-Endpoints wiederum beinhalten Methoden, die ihrerseits entsprechende Methoden des Anwendungskerns aufrufen.
+Die Methoden der Service-Endpoints nehmen Transport-Objekte als Argumente entgegen und liefern Transport-Objekte als Ergebnis zurück.
 [[servicelogik]]
 == Servicelogik
 
 Die Servicelogik gliedert sich, wie der Anwendungskern, in *Fachkomponenten*.
 Die Fachkomponenten setzen im wesentlichen *Service-Schnittstellen* um.
-Sie decken vier wesentliche Aufgaben ab.
+Sie decken fünf wesentliche Aufgaben ab.
+
```

Comment:
Evtl. Aufgaben als Aufzählung / Liste ?

Reply:
Die Formatierung sollte bleiben wie sie war, (nice to have changes wollen wir vermeiden)

Reply:
Formatierung sollte nicht geändert werden. (wir sollten keine nice to have changes durchführen)